active/operators/bort/generate_job_id.py

```diff
@@ -44,30 +46,36 @@ def _insert_bort_scheduled(run_id, update_date, job_start_date, **kwargs):
     :param update_date:  bortdate from the af_bort_runs_details table where bortstatus == complete
     """
     tgt_insert_stmt = "INSERT INTO {table}" \
-                      "({run_id}, job_status, job_start, bortdate) " \
+                      "({run_id}, job_status, job_start, bort_date) " \
                       "VALUES (%s, %s, %s, %s)".format(table=common_variables.AF6_RUNS,
                                                        run_id=common_variables.AF6_RUNS_ID)
     common_hooks.AIRFLOW_NLP_DB.run(tgt_insert_stmt,
                               parameters=(run_id, common_variables.JOB_RUNNING, job_start_date, update_date))
 
     return
 
-def _get_blobs_since_date(date, job_state, **kwargs):
-  """
-  :param date:
-  :param job_state:
-  """
-
 def _get_last_bort_run_id(**kwargs):
     tgt_select_stmt = "SELECT max({run_id}) FROM {table}".format(table=common_variables.AF6_RUNS,
                                                                  run_id=common_variables.AF6_RUNS_ID)
     last_run_id = (common_hooks.AIRFLOW_NLP_DB.get_first(tgt_select_stmt) or (None,))
 
     return last_run_id[0]
 
+def _get_blobs_since_date(date, job_state, **kwargs):
+    """
+    :param date: last resynth task complete date
+    :param state: resynth job complete
+    """
+    tgt_update_stmt = "SELECT distinct hdcorcablobid, ner_date " \
+                      "FROM {table} " \
+                      "WHERE ner_date >= %s " \
+                      "AND ner_status = %s ".format(table=common_variables.AF5_RUNS_DETAILS)
+    return common_hooks.AIRFLOW_NLP_DB.get_records(tgt_update_stmt, parameters=(date, job_state))
+
+
 
 def _get_last_bort_update_date(**kwargs):
-    tgt_select_stmt = "SELECT max(bortdate) " \
+    tgt_select_stmt = "SELECT max(bort_date) " \
```

Comment:
do we need to select max dates? is the combination of NER JOB = COMPLETE and record not in BORT table/BORT JOB != COMPLETE sufficient?

Reply:
it's what we do elsewhere (eg: af5), with the 'moving wave' strat.

Reply:
let's discuss the date filter at standup tomorrow